src/extension.ts

```diff
@@ -260,7 +261,7 @@ async function pathToBinary(): Promise<string> {
 	if (!_pathToBinaryPromise) {
 		let command: string = config('terraform').get('languageServer.pathToBinary');
 		if (!command) { // Skip install/upgrade if user has set custom binary path
-			const installDir = `${extensionPath}/lsp`;
+			const installDir = path.join(extensionPath, '/lsp');
```

Comment:
The documentation doesn't seem very clear on how it handles such case (except that it "allows" it), but shouldn't the platform-dependent separators be now omitted? i.e. shouldn't `/lsp` be just `lsp`?

Reply:
My impression was that it would normalize the separator as well, but it works without the slash so I'll go with that.